api/src/main/java/org/openmrs/module/dhisconnector/api/util/DHISConnectorUtil.java

```diff
@@ -56,6 +50,32 @@ public static String zipMappingBundle (String tempDirectory, List<File> fileList
         return path;
     }
 
+    public static  List<File> unzipMappingBundle (ZipFile zipFile, String tempDirectory) throws IOException {
+        List<File> files = new ArrayList<>();
+        for (Enumeration<? extends ZipEntry> e = zipFile.entries(); e.hasMoreElements();) {
+            ZipEntry zipEntry = e.nextElement();
+            File output = new File(tempDirectory + zipEntry.getName());
+            if (!output.exists()) {
+                output.getParentFile().mkdirs();
+                output.createNewFile();
+            }
+            BufferedInputStream inputStream = null;
+            BufferedOutputStream outputStream = null;
+            try {
+                inputStream = new BufferedInputStream(zipFile.getInputStream(zipEntry));
+                outputStream = new BufferedOutputStream(new FileOutputStream(output));
+                IOUtils.copy(inputStream, outputStream);
+            } catch (IOException exception) {
+                exception.printStackTrace();
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotprintout

Reply:
On it, sorry for the clumsiness

Reply:
Resolved with the commit a3528e0 please review